### ISSUE TITLE
Presenting outcomes at BOSC 2015

### ISSUE THREAD

**Issue by hlapp** [NESCent/r-popgen-hackathon#17]
The [2015 Bioinformatics Open Source Conference](http://www.open-bio.org/wiki/BOSC_2015) takes place in Dublin, Ireland, on July 10-11. Is anyone from the different teams planning to attend that conference? It would be a good venue to present and promote outcomes / products from teams. Abstract submission closes April 3.

Perhaps coordinate plans through this tracker issue?

**Comment by thibautjombart** [comment]
Good point! I did not plan on going, but that may be a good way for the Europeans to catch up?
